ILRepack/ILRepack.csproj

```diff
@@ -10,9 +10,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ILRepacking</RootNamespace>
     <AssemblyName>ILRepack</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
```

Comment:
Most definitely it was a configuration issue with ncrunch. I just couldn't set Cecil to v3.5 from v4.0. It would not let me change it. Will give it a try later today. Or remove it from the commit since it's not really needed.